config/routes.rb

```diff
@@ -26,6 +26,16 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  resources :ex_links
+
+  namespace :admin do
+    get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+    get 'ex_links/check_dead' => 'ex_links#check_dead'
+    get 'ex_links/check_expired' => 'ex_links#check_expired'
+    resources :ex_links
```

Comment:
I am getting

```
ActionController::UrlGenerationError at /admin/ex_links
No route matches {:action=>"del_unused_tags", :controller=>"admin/ex_links"}
```

when changing it to proposed format.

When changing type of action, I am getting:

```
ActionController::RoutingError at /admin/ex_links/del_unused_tags
not found
```